gen.go

```diff
@@ -1386,12 +1486,10 @@ func (g Gen) emitCborUnmarshalSliceField(w io.Writer, f Field) error {
 		return fmt.Errorf("do not yet support slices of %s yet", e.Elem())
 	}
 
-	if err := g.doTemplate(w, f, `
-	{{ if .PreserveNil }}
+	if err := g.doTemplate(w, f, `{{ if .PreserveNil }}
```

Comment:
The formatting changes are making it a bit hard to tell what I should look at.

Reply:
I've undone the whitespace changes, I'll live with the OCD.

Reply:
Let's just do them in a followup PR, this PR was just getting rather large.